app/domain/logs.rb

```diff
@@ -449,12 +449,50 @@ module AuthnJwt
         code: "CONJ00088D"
       )
 
-
-
       ExtractTokenString = ::Util::TrackableLogMessageClass.new(
         msg: "Extract JWT token from request",
         code: "CONJ00089D"
       )
+
+      ValidatingJwtStatusConfiguration = ::Util::TrackableLogMessageClass.new(
+        msg: "Validating JWT status configuration...",
+        code: "CONJ00090D"
+      )
+
+      ValidatedUserHasAccessToStatusWebservice = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully validated user has access to status webservice",
+        code: "CONJ00091D"
+      )
+
+      ValidatedAuthenticatorWebServiceExists = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully validated status webservice exists",
+        code: "CONJ00092D"
+      )
+
+      ValidatedStatusWebserviceIsWhitelisted = ::Util::TrackableLogMessageClass.new(
+        msg: "Successfully status webservice is whitelisted",
```

Comment:
"Successfully **validated** statys webservice is **allowlisted**"

(I think you left out "validated"
Whitelisted is not allowed anymore.... not pilotically correct

Reply:
I was was wondering about it, as it wasn't in scope yet I was sure about changing this, but I have to agree with you we should add things that we will need to change later.